package.json

```
@@ -120,7 +120,7 @@
     "@crazyfactory/tinka": "^1.8.2",
     "@material-ui/core": "^3.9.2",
     "@material-ui/icons": "^3.0.2",
-    "@sentry/browser": "^4.6.1",
+    "@sentry/browser": "^5.0.0",
```

```suggestion
    "@sentry/browser": "5.0.0",
```
testing suggestion feature for github